.github/ISSUE_TEMPLATE/thank_streamlit_team.yml

```diff
@@ -0,0 +1,10 @@
+name: ✨ Thank the Streamlit team!
```

Comment:
nit: We already use the same emoji for the feature template, so maybe we can use something different, e.g. 🙏

<img width="995" alt="image" src="https://github.com/user-attachments/assets/69edb639-ff9f-4b92-9c98-9e00c5048792">

Reply:
Good callout! Let me switch that to some 🙏